scripts/mixins/actuallyadditions.zs

```diff
@@ -74,11 +88,64 @@ zenClass MixinLensMining {
     #    at: {
     #        value: "INVOKE",
     #        target: "Lde/ellpeck/actuallyadditions/api/ActuallyAdditionsAPI;addMiningLensNetherOre(Ljava/lang/String;I)V"
+    #    },
+    #    slice: {
+    #       to: {
+    #           value: "INVOKE",
+    #           target: "Lde/ellpeck/actuallyadditions/mod/config/values/ConfigStringListValues;getValue()[Ljava/lang/String;"
+    #       }
     #    }
     #}
     function removeRegisteredNetherOres(name as string, weight as int) as void {
         // NO-OP
     }
+
+    #mixin Redirect
+    #{
+    #    method: "invoke",
+    #    at: {
+    #        value: "FIELD",
+    #        target: "Lde/ellpeck/actuallyadditions/api/ActuallyAdditionsAPI;STONE_ORES:Ljava/util/List;",
+    #        opcode: 178
+    #    }
+    #}
+    function removeStoneTransformation() as [string] {
+        return null;
+    }
+
+    #mixin ModifyVariable
+    #{
+    #    method: "invoke",
+    #    at: {
+    #       value: "JUMP",
+    #       opcode: 198,
+    #       ordinal: 0
+    #    },
+    #    name: "ores"
+    #}
+    function setTransformationOresIfBlockIsEndstone(ores as [WeightedOre], hitState as IBlockState, hitPos as BlockPos, tile as IAtomicReconstructor) as [WeightedOre] {
```

Comment:
Seems like the fix is not working (or im testing it wrongly).
https://github.com/user-attachments/assets/cec03199-4db6-44d1-81c0-f1918da082f9

Let me describe what mining lens should do.
Originally, Mining Lens turn Stone or Netherrack into respective ores, with various energy use, weights and config lists.
I actually dont need anything of this in E2EE. We only need for Mining Lens to turn **End Stone into End Ores using RF**, thats it. That means, configuration can be skipped and handled inside ZenScript instead of AA config files.

So, all i need is to laser lens trun End Stone into this list of ores with respective weight probabilities:
```
oreEndQuartz - 64
oreEndAluminum - 40
oreEndIron - 30
oreEndCobalt - 20
oreEndArdite - 20
oreEndDiamond - 10
```
![image](https://github.com/user-attachments/assets/cd7d34d3-f767-4f71-876f-7b481139bbd2)

Reply:
Mining Lens is working fine now, great!